src/saml2/saml.py

```diff
@@ -496,6 +500,7 @@ def __init__(self,
                           text=text,
                           extension_elements=extension_elements,
                           extension_attributes=extension_attributes)
+        self.key_info = key_info
```

Comment:
`KeyInfo` is not part of the `SubjectConfirmationDataType` spec. It is only one of the `xs:any` elements that are allowed to extend the `SubjectConfirmationData` element. Extensions to elements are part of `extension_elements`.

Reply:
Thanks. Done.